public/video-ui/src/components/Video/Display.js

```diff
@@ -28,14 +29,38 @@ class VideoDisplay extends React.Component {
     }
   }
 
+  canonicalVideoPageExists = () => {
+    return this.props.usages.totalVideoPages > 0;
```

Comment:
we do this in a few places, should we add it to [video utils](https://github.com/guardian/media-atom-maker/blob/master/public/video-ui/src/util/video.js) so its shared?